Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -110,7 +113,8 @@ import BraintreeCore
         userAction: BTPayPalRequestUserAction = .none,
         offerPayLater: Bool = false,
         currencyCode: String? = nil,
-        requestBillingAgreement: Bool = false
+        requestBillingAgreement: Bool = false,
+        contactPreference: BTContactPreference? = nil
```

Comment:
I think we will need a `.none` default for Obj-C support similar to `BTPayPalRequestUserAction` above